bundles/org.openhab.binding.groheondus/src/main/java/org/openhab/binding/groheondus/internal/handler/GroheOndusBlueHandler.java

```diff
@@ -0,0 +1,150 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.groheondus.internal.handler;
+
+import static org.openhab.binding.groheondus.internal.GroheOndusBindingConstants.*;
+
+import java.io.IOException;
+import java.util.Optional;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.eclipse.smarthome.core.library.types.OnOffType;
+import org.eclipse.smarthome.core.library.types.StringType;
+import org.eclipse.smarthome.core.thing.ChannelUID;
+import org.eclipse.smarthome.core.thing.Thing;
+import org.eclipse.smarthome.core.thing.ThingStatus;
+import org.eclipse.smarthome.core.thing.ThingStatusDetail;
+import org.eclipse.smarthome.core.types.Command;
+import org.eclipse.smarthome.core.types.State;
+import org.grohe.ondus.api.OndusService;
+import org.grohe.ondus.api.model.BaseApplianceCommand;
+import org.grohe.ondus.api.model.blue.Appliance;
+import org.grohe.ondus.api.model.blue.ApplianceCommand;
+import org.grohe.ondus.api.model.blue.TapType;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * @author Florian Schmidt - Initial contribution
+ */
+@NonNullByDefault
+public class GroheOndusBlueHandler<T, M> extends GroheOndusBaseHandler<Appliance, ApplianceCommand> {
```

Comment:
Are the `<T, M>` generics even used?

Reply:
Good catch :) No, it's not used, so it can be removed :)